mongodb-advanced/components/people-component.js

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default {
+  controller: Ctrl,
+  templateUrl: 'mongodb-advanced/people-component.html'
+};
+
+function Ctrl($scope, exPeopleService) {
+  const self = this;
+  self.showMyModal = false;
+
+  exPeopleService.getPeople(people => {
```

Comment:
This initialization should occur inside an $onInit function as described here: https://scotch.io/tutorials/understanding-angular-1-5-lifecycle-hooks#toc-oninit

Reply:
I meant to ask you about this one. I tired to do that originally as having duplicate code annoys the heck out of me. But for some reason could not make it work. Since it is working now I suspect that I had something else incorrect and not this part of the coding.